node/.eslintrc.js

```diff
@@ -59,7 +64,7 @@ const eslintConfig =
 			}
 		],
 		'keyword-spacing'      : 2,
-		'linebreak-style'      : [ 2, 'unix' ],
+		'linebreak-style'      : [ 2, isWindows ? 'windows' : 'unix' ],
```

Comment:
Why would it depend on OS thought? This will mean people contributing from different operating systems having different code style that either passes Windows CI or *nix CI.

Reply:
This was for CI lint pass in Windows.

Reply:
Then it doesn't really make sense to me. Why would we want to check for different line breaks depending on OS check is running on?

Reply:
Pretty sure that line endings change in place from LF to CRLF when fetching the git repo in Windows. AFAIR that could be a feature of git. Probably it can be disabled.

Reply:
Hm, I recall something like that too... Okay, thanks for clarifying.

Reply:
Here: https://stackoverflow.com/questions/21822650/disable-git-eol-conversions

AFAIU it's a local setting so we cannot force the Windows machine in GH CI to behave as we would wish.

Reply:
I see, it is fine then, thanks!